app/src/org/commcare/activities/RootMenuHomeActivity.java

```diff
@@ -38,7 +38,6 @@ public void onCreateSessionSafe(Bundle savedInstanceState) {
     @Override
     public void onResumeSessionSafe() {
         super.onResumeSessionSafe();
-        navDrawerController.reopenDrawerIfNeeded();
```

Comment:
not related to this PR but I realized this was causing undesirable behavior while testing